messages.go

```diff
@@ -2,20 +2,21 @@ package validate
 
 // Messages for the checkers; this can be changed for i18n.
 var (
-	MessageRequired   = "must be set"
-	MessageDomain     = "must be a valid domain"
-	MessageURL        = "must be a valid url"
-	MessageEmail      = "must be a valid email address"
-	MessageIPv4       = "must be a valid IPv4 address"
-	MessageHexColor   = "must be a valid color code"
-	MessageLenLonger  = "must be longer than %d characters"
-	MessageLenShorter = "must be shorter than %d characters"
-	MessageExclude    = "cannot be ‘%s’"
-	MessageInclude    = "must be one of ‘%s’"
-	MessageInteger    = "must be a whole number"
-	MessageBool       = "must be a boolean"
-	MessageDate       = "must be a date as ‘%s’"
-	MessagePhone      = "must be a valid phone number"
+	MessageRequired       = "must be set"
+	MessageDomain         = "must be a valid domain"
+	MessageURL            = "must be a valid url"
+	MessageEmail          = "must be a valid email address"
+	MessageIPv4           = "must be a valid IPv4 address"
+	MessageHexColor       = "must be a valid color code"
+	MessageLenLonger      = "must be longer than %d"
+	MessageLenShorter     = "must be shorter than %d"
```

Comment:
These error messages will not look confusing:

    name: must be longer than 15

Instead of the previous:

    name: must be longer than 15 characters

Even when passing a list this now looks confusing, for example if something needs at least 2 tags:

    tags: must be longer than 2

Reply:
We could rephrase to something more specific and type agnostic, like:

```
tags: length must the greater than 2
```

We could also use specific messages for each type.

If this is a blocker for this feature, we can also leave this validation as it was and create two new ones: `GreaterThan()` and `LesserThan()`.

WDYT?

Reply:
Why write error messages, tho?
It's far better to have error code (or the validation that was run) in every scenario i can think

```golang
   type LenErr struct {
      Min int64
      Max int64
   }
```

that way everyone using the error can and must react appropriate instead of just dumping into the user interface.

Probably a big change anyways. My comment is useless